roles/lib_openshift/src/class/oc_project.py

```diff
@@ -0,0 +1,183 @@
+# pylint: skip-file
+# flake8: noqa
+
+
+# pylint: disable=too-many-instance-attributes
+class OCProject(OpenShiftCLI):
+    ''' Class to wrap the oc command line tools '''
+    kind = 'namespace'
```

Comment:
https://github.com/openshift/origin/pull/11647 is the pull - it is stuck in limbo waiting on a review from @deads2k and @liggitt